### ISSUE TITLE
Calendar year selection needs a visual choosing method

### ISSUE THREAD

**Issue by BenRacicot** [angular/material2#6587]
#### Bug, feature request, or proposal:
feature request

#### What is the expected behavior?
When a user clicks on the calendar icon the calendar popup appears and displays years to choose from so that a user does not have to click through a list of years.

#### What is the current behavior?
When a user clicks on the calendar icon the calendar popup appears. However, it seems that in order to change the year you must click the < and > to choose a year. Maybe I'm missing something but this is a most undesirable UX.

#### What are the steps to reproduce?
```
<md-input-container>
    <input mdInput [mdDatepicker]="birthDatePicker" formControlName="birthDate" [max]="maxDate">
    <button mdSuffix [mdDatepickerToggle]="birthDatePicker"></button>
</md-input-container>
<md-datepicker #birthDatePicker startView="year"></md-datepicker>

```
#### What is the use-case or motivation for changing an existing behavior?
Users will not click until they hit a far off year, such as birthdate.

**Comment by willshowell** [comment]
See https://github.com/angular/material2/issues/5845

**Comment by BenRacicot** [comment]
Closed for #5845 thanks @willshowell
